superset/assets/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -108,6 +109,34 @@ class HeaderActionsDropdown extends React.PureComponent {
     this.props.startPeriodicRender(refreshInterval * 1000);
   }
 
+  downloadChartImage(ev){
+    //debugger;
+    var jtarget = $(ev.currentTarget)
+                .parents('.dashboard');
+    window.setTimeout(() => {
+      // fix transparent backgrounds
+      var bgc = jtarget.css('background-color');
+      var bgcs = [bgc].concat(
+                    jtarget.parents().map((i, el) => $(el).css('background-color')).toArray()
+                  )
+                  .filter(c => c !== "rgba(0, 0, 0, 0)");    
+      var node = jtarget.css('background-color', bgcs[0]||bgc);
+      var title = jtarget.find('.dashboard-header .editable-title input').val();
+      
+      domtoimage.toJpeg(node[0], { quality: 0.95 })
+        .then(function (dataUrl) {
+            var link = document.createElement('a');
+            link.download = title + '.jpeg';
+            link.href = dataUrl;
+            link.click();
+        })
+        .finally(function(){
+          node.css('background-color', bgc);
```

Comment:
Instead of modifying the dom node in place then switching the colors back, is there a reason why we wouldn't just want to copy the node, do all your modifications to it, then delete the node afterwards?